localplanning.services/astro.config.mjs

```diff
@@ -1,19 +1,21 @@
-// @ts-check
 import { defineConfig, envField, fontProviders } from "astro/config";
 import react from "@astrojs/react";
 import tailwindcss from "@tailwindcss/vite";
-
+import { loadEnv } from "vite";
 import icon from "astro-icon";
 
+const { PUBLIC_LPS_URL, MODE } = loadEnv(process.env.NODE_ENV, process.cwd(), "");
+
 // https://astro.build/config
 export default defineConfig({
   integrations: [react(), icon()],
-  site: "https://localplanning.services",
+  site: PUBLIC_LPS_URL,
```

Comment:
Setting this will now generate sitemaps, which bumps up our Lighthouse score a fair bit 💹 

Docs: https://docs.astro.build/en/reference/configuration-reference/#site